lib/pbench/server/auth/__init__.py

```diff
@@ -0,0 +1,470 @@
+from flask_restful import abort
+from http import HTTPStatus
+import jwt
+import logging
+import requests
+from requests.structures import CaseInsensitiveDict
+from typing import Dict, List, Optional, Union
+from urllib.parse import urljoin
```

Comment:
This should be two blocks:
```
from http import HTTPStatus
import logging
from typing import Dict, List, Optional, Union
from urllib.parse import urljoin

from flask_restful import abort
import jwt
import requests
from requests.structures import CaseInsensitiveDict
```

Reply:
Okay so the first block belongs to the standard python library packages

Reply:
Yep:

1. First block:  Python standard library packages
2. Second block:  Third-party library packages
3. Third block:  Project library packages

I admit, when we first started implementing this convention, it felt kind of arbitrary and pointless, but then I went and did a bunch of work on packaging and installation, and it's actually kind of helpful that the third-party dependencies aren't hidden among the standard ones that we get "for free".

Reply:
I stumbled across this python package [isort](https://pypi.org/project/isort/). We give the python file as a parameter and it sorts the import. I think they are worth taking a look

Reply:
`vscode` has an isort plugin; I tried it out, and it seems to work ... except that it doesn't separate *our* third party imports from other third party imports. (E.g., `sqlalchemy` comes after `pbench`.) I'm not sure if there's a way to configure our traditional third grouping with the intervening blank line.